### PR TITLE
New guidelines for developpers

### PR DESCRIPTION
**Pull request by Gab-D-G**

### CODE REVIEW

**Comment by Gab-D-G** [comment]
@gdevenyi please review contributing.md

File: docs/conf.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
 author = 'CoBrALab'
 
 # The full version, including alpha/beta/rc tags
-release = '0.4.6'
```
Comment (gdevenyi) [inline docs/conf.py:25]: This should be a seperate commit

File: docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+## Instructions to create a pull request
+
+1. On github, fork the RABIES repository to have your own copy. 
+2. Clone your repository to carry out local modifications and testing. Use the `--recursive` option to download the submodules together with the main RABIES package.
+3. Make your modifications to the code.
+4. Testing and debugging: install your updated version of the package with ```python setup.py install```, using a proper dev environment (see above). Your can test the workflow with specific parameters by editing the ```debug_workflow.py``` script, and executing in debug mode with Spyder (see below). Before commiting changes, make sure that running ```error_check_rabies.py``` completes with no error.
+5. Commit and push your modifications to Github, and create a pull request from your forked repo to the original.
```
Comment (gdevenyi) [inline docs/contributing.md:25]: You need to mention making a branch

File: docs/requirements.txt [hunk 3]
```diff
@@ -5,4 +5,4 @@ sphinx-rtd-theme
 sphinxcontrib-bibtex
 sphinxcontrib-programoutput==0.17
 jinja2==3.1.1
-rabies==0.4.7
```
Comment (gdevenyi) [inline docs/requirements.txt:8]: this should have a seperate commit with the version changes above

File: rabies/preprocess_pkg/utils.py [hunk 4]
```diff
@@ -1,7 +1,3 @@
-import os
-import pathlib
-import SimpleITK as sitk
-import numpy as np
```
Comment (gdevenyi) [inline rabies/preprocess_pkg/utils.py:4]: This is another separate commit

**Review by gdevenyi — CHANGES_REQUESTED** [review]

**Comment by Gab-D-G** [comment]
This branch is replaced by the documentation branch #331 , where the commit history is in line with @gdevenyi's feedback. The pull request will be closed.

**Pull request closed without merging**